src/sidebar.js

```diff
@@ -90,13 +90,28 @@ module.exports = {
             'guides/customize-identity-schema',
             {
               type: 'category',
-              label: 'Enabling Social Sign-in',
+              label: 'Social Sign-in',
               items: [
-                'guides/social-signin/google',
-                'guides/social-signin/facebook',
-                'guides/social-signin/microsoft',
-                'guides/social-signin/github',
-                'guides/social-signin/generic'
+                'guides/social-signin/overview',
+                {
+                'Integrating Providers': [
+                  'guides/social-signin/google',
+                  'guides/social-signin/facebook',
+                  'guides/social-signin/microsoft',
+                  'guides/social-signin/github',
+                  'guides/social-signin/generic',
+                  'guides/social-signin/apple',
+                  'guides/social-signin/gitlab',
+                  'guides/social-signin/auth0',
+                  'guides/social-signin/slack',
+                  'guides/social-signin/spotify',
+                  'guides/social-signin/discord',
+                  'guides/social-signin/twitch',
+                  'guides/social-signin/netid',
```

Comment:
it does not exists and build fails with : 

```
 Starting the development server...
[ERROR] Loading of version failed for version current
[ERROR] Error: Invalid sidebar file at "src/sidebar.js".
These sidebar document ids do not exist:
- guides/social-signin/netid
```

Reply:
That's fixed, the netid doc was on my machine but I didn't commit it 😅